Simplenote/en.lproj/Localizable.strings

```diff
@@ -1,3 +1,6 @@
+/* Presented when there are multiple selected notes */
+"%d Selected" = "%d Selected";
```

Comment:
This is the fix to the issue raised [here](https://github.com/Automattic/simplenote-macos/pull/1004#discussion_r677137522) 👍 

Thank you @oguzkocer @jleandroperez